<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SPT-CPU-0330" name="cec_regression_callcenter_testcases.CC_VerifyCallCenterManagerSuccessfullyConfigureNewDiscretionaryReward" tests="1" failures="0" timestamp="01 Jun 2018 13:01:21 GMT" time="13.870" errors="1">
  <testcase name="logIn" time="13.870" classname="cec_regression_callcenter_testcases.CC_VerifyCallCenterManagerSuccessfullyConfigureNewDiscretionaryReward">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerExceptionat org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)at com.sun.proxy.$Proxy7.click(Unknown Source)at cec_callcenter_pageobjects.DiscretionaryReward.clickpromotionlink(DiscretionaryReward.java:28)at cec_regression_callcenter_testcases.CC_VerifyCallCenterManagerSuccessfullyConfigureNewDiscretionaryReward.logIn(CC_VerifyCallCenterManagerSuccessfullyConfigureNewDiscretionaryReward.java:26)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- logIn -->
</testsuite> <!-- cec_regression_callcenter_testcases.CC_VerifyCallCenterManagerSuccessfullyConfigureNewDiscretionaryReward -->
